crates/goose-mcp/src/developer/mod.rs

```diff
@@ -45,6 +46,14 @@ use xcap::{Monitor, Window};
 
 use ignore::gitignore::{Gitignore, GitignoreBuilder};
 
+/// Configuration for the developer extension
```

Comment:
can we leave out this for now? we're overhauling the whole settings thing. just keep it simple and pass around a boolean disable goosehints I would say

Reply:
yeah, will do